crates/iota-json-rpc-types/src/iota_event.rs

```diff
@@ -50,17 +50,94 @@ pub struct IotaEvent {
     pub type_: StructTag,
     /// Parsed json value of the event
     pub parsed_json: Value,
-    #[serde_as(as = "Base58")]
-    #[schemars(with = "Base58")]
-    /// Base 58 encoded bcs bytes of the move event
-    pub bcs: Vec<u8>,
+    #[schemars(with = "Base64")]
```

Comment:
Is this needed? In the upstream main branch this doesn't exist

Reply:
Let me check! Thanks for pointing it out